asreview/datasets.py

```diff
@@ -300,77 +292,103 @@ def find(self, dataset_name):
-        latest_only: bool
-            Only include the latest version of the dataset.
+        include: str, iterable
+            List of groups to include
+        exclude: str, iterable
+            List of groups to exclude from all groups.
+        serialize: bool
+            Make returned list serializable.
         raise_on_error: bool
             Raise error when entry point can't be loaded.
 
         Returns
         -------
-        dict:
-            Dictionary with group names as keys and lists of datasets as
-            values.
+        list:
+            List with datasets as values.
         """
-        if group_name is None:
-            group_names = self.groups
-        elif not is_iterable(group_name):
-            group_names = [group_name]
+
+        if include is not None and exclude is not None:
+            raise ValueError("Cannot exclude groups when include is not None.")
+
+        if include is not None:
+            if not is_iterable(include):
+                include = [include]
+            groups = include
+        elif exclude is not None:
+            if not is_iterable(exclude):
+                exclude = [exclude]
+            groups = self.groups.copy()
+            for group_id in exclude:
+                try:
+                    groups.remove(group_id)
+                except ValueError:
+                    pass
         else:
-            group_names = group_name
+            groups = self.groups.copy()
 
         dataset_groups = get_entry_points('asreview.datasets')
 
-        dataset_list = {}
-        for group in group_names:
+        group_list = []
+        for group in groups:
             try:
-                dataset_list[group] = \
-                    dataset_groups[group].load()().list(latest_only=latest_only)
+                group_list.append(dataset_groups[group].load()())
             except Exception as err:
 
                 # don't raise error on loading entry point
                 if raise_on_error:
                     raise err
 
-        return dataset_list
+        if serialize:
+            dataset_list_ser = []
+            for data_group in group_list:
+                try:
+                    group_ser = []
+                    for dataset in data_group.datasets:
+                        group_ser.append(dataset.__dict__())
+                    dataset_list_ser.append({
+                        "group_id": data_group.group_id,
+                        "description": data_group.description,
+                        "datasets": group_ser
+                    })
+                except Exception as err:
+                    # don't raise error on loading entry point
+                    if raise_on_error:
+                        raise err
+
+            return dataset_list_ser
+
+        return group_list
 
 
 class BenchmarkDataGroup(BaseDataGroup):
     group_id = "benchmark"
     description = "A collections of labeled datasets for benchmarking."
```

Comment:
What is a good description (displayed in the GUI)?

Reply:
It might not be necessary to display the description of non-featured datasets. Such description is repeated with the text in the GUI.

Reply:
I removed the text in the GUI. This might be very useful for the extension data tab. It distinguishes the datasets from various plugins.

Reply:
Good point. I added text for extension datasets.